webnn/gpu/instance_normalization.https.any.js

```diff
@@ -0,0 +1,10 @@
+// META: title=test WebNN API instanceNormalization operation
+// META: global=window,dedicatedworker
+// META: script=../resources/utils.js
+// META: timeout=long
+
+'use strict';
+
+// https://webmachinelearning.github.io/webnn/#api-mlgraphbuilder-instancenorm
+
+testWebNNOperation('instanceNormalization', buildLayerNorm, 'gpu');
```

Comment:
Here **buildLayerNorm** function of https://github.com/web-platform-tests/wpt/pull/43684 can be reused.